api/client/service/update.go

```diff
@@ -33,6 +34,19 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.StringSlice("command", []string{}, "Service command")
 	flags.StringSlice("arg", []string{}, "Service command args")
 	addServiceFlags(cmd, opts)
+
+	flags.StringSlice(flagEnvRemove, []string{}, "Remove an environment variable")
+	flags.StringSlice(flagLabelRemove, []string{}, "Remove a label by its key")
+	flags.StringSlice(flagMountRemove, []string{}, "Remove a mount by its target path")
+	flags.StringSlice(flagPublishRemove, []string{}, "Remove a published port by its target port")
+	flags.StringSlice(flagNetworkRemove, []string{}, "Remove a network by name")
+	flags.StringSlice(flagConstraintRemove, []string{}, "Remove a constraint")
+	flags.VarP(&opts.labels, flagLabelAdd, "l", "Add or update service labels")
```

Comment:
Not sure how I feel about the shorthand options for add being the same as `service create` but the long-form versions having the `-add` suffix. Any thoughts?